Nautilus/Handlers/CustomSoundHandler.cs

```diff
@@ -19,11 +19,12 @@ public static class CustomSoundHandler
     /// <param name="id">The Id of your custom sound which is used when checking which sounds to play.</param>
     /// <param name="filePath">The file path on disk of the sound file to load.</param>
     /// <param name="busPath">The bus path to play the sound on.</param>
+    /// <param name="mode">The audio MODE of the sound.</param>
     /// <returns>the <see cref="Sound"/> loaded</returns>
-    public static Sound RegisterCustomSound(string id, string filePath, string busPath)
+    public static Sound RegisterCustomSound(string id, string filePath, string busPath, MODE mode = MODE.DEFAULT)
```

Comment:
Mark this as a breaking change in the description, please. You are adding a new parameter